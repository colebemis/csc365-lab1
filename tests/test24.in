R: grade=6 bus=55 teacher=kerbs
Q