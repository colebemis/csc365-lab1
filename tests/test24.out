Enter query: 17472e443362a4534c33; 3.1; 6; KERBS, BENITO; 55
d45c6f546f9cb18f9828; 2.93; 6; KERBS, BENITO; 55
Enter query: 